docs/android/linter.md

```diff
@@ -0,0 +1,137 @@
+---
+title: "Android Linter"
+sidebar_label: "Linter"
+---
```

Comment:
Didn't we just talk about this in the codestyle?

Reply:
Here I want to go in more details, especially writing custom lints rules or adapt some rules to our project.

Reply:
But then why are we explaining KTLint twice

Reply:
It's because KTLint is a linter at first but it also have a plugin that we use to format the code (that's why it's in codestyle)